examples/S15_python/change_env.py

```diff
@@ -47,7 +47,7 @@ def build_model_smoldyn():
     decay = s.addReaction("decay", subs=[trans], prds=[], rate=math.log(2) / 20e-3)
 
     # BOUTON
-    path = smoldyn.Path2D((1000, 0), (1000, 1000), (0, 1000), (0, 0))
+    path = s.addPath2D((1000, 0), (1000, 1000), (0, 1000), (0, 0))
```

Comment:
`smoldyn.Path2d` now requires a `Simulation` object as one of its arguments. Users are encouraged to use `Simualtion.addPath2d` which forwards its `Simulation` object to `Path2D` automatically.